CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+
+6.0.0-beta.5 / YYYY-MM-DD
+==================
+
+## Possible breaking changes
+  * [[PR #211]](https://github.com/inuitcss/inuitcss/pull/211) - Rename tables objects: use singular
+
+## Other fixes or features
+  * [[PR #229]](https://github.com/inuitcss/inuitcss/pull/229) - Create this changelog file
+  * [[PR #215]](https://github.com/inuitcss/inuitcss/pull/215) and [[PR #216]](https://github.com/inuitcss/inuitcss/pull/216) - Use simpler value for example global transitions variable
+  * [[PR #213]](https://github.com/inuitcss/inuitcss/pull/213) - add some rules to the `inuit-hidden-visually` mixin
+  * [[PR #209]](https://github.com/inuitcss/inuitcss/pull/209) - Fix typo in the title banner in `objects.wrapper`
```

Comment:
Hum... I think the line is a bit too blurry to make a good distinction, but I can give it a shot if you guys think the added value is worth it.